packages/testing/config/jest/api/jest.setup.js

```diff
@@ -218,9 +218,26 @@ const wasDbUsed = () => {
   }
 }
 
+beforeEach(() => {
+  // Attempt to emulate the request context isolation behavior
+  const fakeContextStore = new Map()
```

Comment:
Just a personal preference for the term "mock" or "mocked" rather than "fake". It's real for the purposes of a test from its perspective.